src/main/scala/io/iohk/praos/domain/SlotInEpochCalculator.scala

```diff
@@ -0,0 +1,18 @@
+package io.iohk.praos.domain
+
+class SlotInEpochCalculator(epochLength: Int, slotDurationInMilliseconds: Int) {
+
+  def calculateSlotNumber(timeProvider: TimeProvider): Int =
+    ((timeProvider.getTime - timeProvider.initialTime) / slotDurationInMilliseconds) + 1
+
+  def calculateEpochNumber(timeProvider: TimeProvider): Int =
```

Comment:
Minor detail: Maybe we could add here and in `calculateSlotNumber` a comment regarding these numbers starting from 1 and not 0?

Reply:
:+1: